lib/node-repo.js

```diff
@@ -4,6 +4,15 @@ const githubClient = require('./github-client')
 
 const resolveLabels = require('./node-labels').resolveLabels
 
+function deferredResolveLabelsThenUpdatePr (options) {
+  if (options.timeoutInSec) {
+    setTimeout(() => resolveLabelsThenUpdatePr(options),
+              options.timeoutInSec * 1000)
+  } else {
+    resolveLabelsThenUpdatePr(options)
```

Comment:
What if we got rid of this `else` block and always executed this async? Something like:

```javascript
const timeoutMillis = (options.timeoutInSec || 0) * 1000

setTimeout(resolveLabelsThenUpdatePr, timeoutMillis, options)
```